subsys/bluetooth/host/Kconfig

```diff
@@ -1188,4 +1188,22 @@ config BT_HCI_VS_EVT_USER
 	  Enable registering a callback for delegating to the user the handling of
 	  VS events that are not known to the stack
 
+config BT_NRPA_LIMIT
+	bool "Limit the number of non-resolvable private addresses (NRPA)"
+	depends on !BT_PRIVACY
+	help
+	  When enabled, the BLE Host will only generate the BT_NRPA_LIST_SIZE
+	  number of NRPAs and will keep using them until the next reboot.
+
+	  WARNING: This will compromise the privacy of the device.
+	  Use of this feature in production is strongly discouraged.
+
+config BT_NRPA_LIST_SIZE
+	int "Number of non-resolvable private addresses generated by the BLE Host"
+	depends on BT_NRPA_LIMIT
+	range 1 100
+	default 10
+	help
+	  Number of non-resolvable private addresses generated by the BLE Host.
```

Comment:
I wonder if we could simplify this by having just one `BT_NRPA_LIST_SIZE` option with `depends on BT_TESTING`. It would have `default 0` which would mean that there is no limit (i.e. existing behaviour).

Also, let's not start introducing the "BLE" term anywhere into documentation. It's not an official SIG term and I think we've succeeded in avoiding it pretty well until now. In the above cases you could refer simply to "host", "Bluetooth host" or "Bluetooth host stack". Private addresses is a Bluetooth LE feature, so there's no need to explicitly underline that.

Reply:
Don't think this is introducing the BLE term when talking about the host, I see it in use already in Zephyr doc.
But I agree we should use Bluetooth host or host, as it is done in the core spec.
Or rather Bluetooth Host or Host (I see the capital letter/H is used in the spec) :)